src/guides/debugging-an-example-smart-contract/index.md

```diff
@@ -196,27 +246,34 @@ We next want to interact with the smart contract to see how it works when workin
 1. To verify that the variable has changed values, run the `get()` function again:
 
    ```javascript
-   SimpleStorage.deployed().then(function(instance){return instance.get.call();}).then(function(value){return value.toNumber()});
+   SimpleStorage.deployed()
+     .then(function (instance) {
+       return instance.get.call();
+     })
+     .then(function (value) {
+       return value.toNumber();
+     });
    ```
 
    The output should look like this:
 
    ```javascript
```

Comment:
```suggestion
   ```
```

Reply:
Did you mean to delete only the open tag here?  I believe commiting this would make the closing ``` turn the rest of the document into a code block.

Reply:
Whoops! I meant to just delete the javascript tag. :)